docs/user-guide/cli-using-global-profile-configuration.md

```diff
@@ -0,0 +1,535 @@
+
+
+## Initializing global configuration
+
+Verify that you have installed [the @next release of Zowe CLI](cli-install-cli-next.md).
+
+You have two options to initialize the global configuration:
+ * Define a connection to z/OSMF (recommended).
+ * Create your configuration files manually.
+
+***Follow these steps:***
+
+1. Issue the command:
+
+   ```
+   zowe config init --global-config
+   ```
+
+   The CLI responds with prompts for a username and password.
+
+2. Respond to the prompts with a username and password for a mainframe service such as z/OSMF.
+
+   **Note:** The `config init` command ensures that your credentials are stored securely on your computer by default.
+
+   After you respond, the `zowe.config.json` global configuration file is added to your local `.zowe` directory. This is the primary location where your MF service connection details such as host and port are defined. Use this configuration file for the following procedures.
+
+3. Edit the `zowe.config.json` file to edit the name of the LPAR,  supply a value for the host name, and edit the port number as needed.
+
+4. Issue a Zowe CLI command to test that you can access z/OSMF. For example, list all data sets under your user ID:
+
+   ```
+   zowe zos-files list data-set "MY.DATASET.*"
+   ```
+
+   A list of data sets is returned. You successfully configured Zowe CLI to access a z/OSMF instance.
+
+   If the CLI returns an error message, verify that you have access to the target system. Examine your configuration files in a text editor to verify that the information you entered is correct.
+
+**Important!:** After the configuration files are in place (either via the `zowe config init` command or by manually creating the files), the now-deprectaed `zowe profiles` commands will no longer function. Zowe CLI will return errors if you attempt to make use of old profiles.
+
+## Initializing user-specific configuration (Optional) 
+
+You can generate a *user-specific* configuration file that overrides the values defined in the global `zowe.config.json` file.
+
+To generate both global (`zowe.config.json`) and user (`zowe.config.user.json`) config files, issue the command `zowe config init --global-config --user-config`.
+
+In your user-specific file, notice that the top level defaults, plugins, and secure fields are empty. The profiles do not have any properties. You can add your connection details as properties here to override properties in `zowe.config.json`, or add add new connections.
+## Editing global configuration
+
+After the initial setup, you can define additional mainframe services to the global (or user-specific) configuration file.
+
+Open the `~/.zowe/zowe.config.json` file in a text editor or IDE on your computer. The profiles object contains connection and other frequently needed information for accessing various services. For example:
+
+```json
```

Comment:
The LPAR1 placeholder is no longer created. The plugins section is no longer created. A base profile IS now created.

You will have to get an new example and some explanation from the development team.